config/labels.yml

```diff
@@ -78,6 +78,11 @@ repos:
     <<: *core_ruby_common
     <<: *release
     <<: *release_d_plus
+  manageiq-api:
+    <<: *core_ruby_common
+    <<: *gems_common
+    <<: *release
+    <<: *release_f_plus
```

Comment:
This part is ok because it is stating to show all backporting labels from f onward.  We might want to change the yaml label to backport_f_and_below or something to be clearer.